tests/unit/#266.test.scala

```diff
@@ -0,0 +1,22 @@
+// SYNTAX TEST "source.scala"
+
+   210
+// ^^^ constant.numeric.scala
+   0b1
+// ^^^ constant.numeric.scala
+   0B1
+// ^^^ constant.numeric.scala
+   0b0001
+// ^^^ constant.numeric.scala
```

Comment:
```suggestion
// ^^^^^^ constant.numeric.scala
```
it doesn't seem to cover the whole number :thinking:

Reply:
I missed that one